docs/book/src/getting-started.md

```diff
@@ -9,32 +9,40 @@
     Optional if you do not have an existing Kubernetes cluster
     - [kind][kind-install]
     - [Docker][docker-install]
-
-2. Set up Apache CloudStack credentials
-    - Create a file named `cloud-config` in the repo's root directory, substituting in your own environment's values
-        ```
-        [Global]
-        api-url = <cloudstackApiUrl>
-        api-key = <cloudstackApiKey>
-        secret-key = <cloudstackSecretKey>
-        ```
-
-    - Run the following command to save the above Apache CloudStack connection info into an environment variable, to be used by clusterctl, where it gets passed to CAPC:
-        ```
-        export CLOUDSTACK_B64ENCODED_SECRET=$(base64 -w0 -i cloud-config)
-        ```
-
-3. Register the capi-compatible templates in your Apache CloudStack installation.
+    
+2. Register the capi-compatible templates in your Apache CloudStack installation.
     - Prebuilt images can be found [here][prebuilt-images]
     - To build a compatible image see [CloudStack CAPI Images][cloudstack-capi-images]
 
-4. Create a management cluster. This can either be :
-    - An existing Kubernetes cluster : For production use-cases a "real" Kubernetes cluster should be used with appropriate backup and DR policies and procedures in place. The Kubernetes cluster must be at least v1.19.1.
+3. Create a management cluster. This can either be :
+    - An existing Kubernetes cluster: For production use-cases a "real" Kubernetes cluster should be used with appropriate backup and DR policies and procedures in place. The Kubernetes cluster must be at least v1.19.1.
 
     - A local cluster created with `kind`, for non production use
         ```
         kind create cluster
         ```
+4. Set up Apache CloudStack credentials as a secret in the management cluster
+   - Create a file named `cloud-config.yaml` in the repo's root directory, substituting in your own environment's values
+       ```
+       apiVersion: v1
+       kind: Secret
+       metadata:
+         name: cloudStackCredentials
```

Comment:
this name needs to be referenced in the corresponding failureDomain in the CloudStackCluster. We should mention that somewhere

Reply:
Secret name *cloudStackCredentials* is a newly added default value for CLOUDSTACK_FD1_NAME in the templates.  No need for the end-user to coordinate these.

Reply:
Changing to cloudstack-credentials because camel case isn't supported (per the below comment).

Reply:
The secret name used here is the newly added default secret name in the templates.  It should all connect without user involvement.